source/stylesheets/refills/_progress-bar-indication.scss

```diff
@@ -21,7 +21,7 @@
   width: 100%;
 
   > span.meter {
-    @include box-sizing(border-box);
+    box-sizing: border-box;
```

Comment:
Properties should be ordered background-color, background-repeat, background-size, border, border-bottom-right-radius, border-radius, border-top-right-radius, box-sizing, display, height, width